101-managed-application/README.md

```diff
@@ -21,10 +21,12 @@ To learn more about Managed Application definitions and Marketplace please visit
 
 ## Exploring created Managed Application
 
-This sample deployment creates the following two resources:
+This sample template combines two steps:
 
-1) A Service catalog managed application definition.
-2) A managed application that contains a storage account.
+1) Deploys a Service catalog managed application definition.
+2) Deploys a managed application using deployed application definition.
+
+You may use deployed application definition to create more managed applications of this sample.
```

Comment:
Suggest more detail here, e.g.  (remove line 29 replace with)

In a production environment a typical scenario would deploy the application definition and application instance as separate steps.  For example, a definition is deployed once and an application many times.  In this sample the two steps are combined to make it easy to quickly see the final result of those steps.